src/main/java/com/cinema_booking/booking/movie/dto/model/MovieRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.cinema_booking.booking.movie.dto.model;
+
+import java.util.UUID;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class MovieRequest {
+    private UUID id;
```

Comment:
Are we not validating it? Still, we don't really need to add id filed here, Hibernate will create unique id on it's own